runtime/drivers/file/repo.go

```diff
@@ -159,6 +179,41 @@ func (c *connection) Delete(ctx context.Context, filePath string, force bool) er
 
 // Sync implements drivers.RepoStore.
 func (c *connection) Sync(ctx context.Context) error {
+	cache := make(map[string][]byte)
+	for _, p := range c.cachedPaths {
+		p = filepath.Join(c.root, p)
+		_, err := os.Stat(p)
+		if err != nil {
+			if os.IsNotExist(err) {
+				continue
+			}
+			return err
+		}
+		err = filepath.Walk(p, func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				return err
+			}
+			if !info.IsDir() {
+				b, err := os.ReadFile(path)
+				if err != nil {
+					return err
+				}
+
+				rel, err := filepath.Rel(c.root, path)
+				if err != nil {
+					return err
+				}
+				cache[strings.TrimLeft(rel, "/")] = b
+			}
+			return nil
+		})
+		if err != nil {
+			return err
+		}
+	}
+	c.cacheMutex.Lock()
+	defer c.cacheMutex.Unlock()
+	c.assetsCache = cache
```

Comment:
The `file` driver does not have a cloning latency like Github has, so there is no need for caching assets in it.